src/DB/projects.json

```
@@ -2703,4 +2703,22 @@
       }
     ]
   }
+  {
+    "github_username": "dhruvishavaghani",
+    "Social_media": {
+      "gitHub": "https://github.com/dhruvishavaghani",
+      "LinkedIn": "http://linkdin-www.linkedin.com/in/dhruvisha-vaghani-0b97b3232",
+      "Instagram": "",
+      "YouTube": "",
+      "Twitter": ""
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/dhruvishavaghani/NewsZilla",
+        "title": "NewsZilla",
+        "description": "NewsZilla is a live news website with all categories of news.",
+        "tech": ["HTML", "CSS", "ReactJs"]
+      }
+    ]
+  },
```

remove the comma `,`
```suggestion
  }
```